mod_reforged/hooks/entity/tactical/humans/militia_veteran.nut

```diff
@@ -46,7 +37,8 @@
 	    		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_formidable_approach"));
 	    	}
 
-    		if (::Reforged.Items.isDuelistValid(weapon))
+	    	local offhand = this.getOffhandItem();
```

Comment:
Why change this away from `isDuelistValid`?

Reply:
Because I only want duelist on the ones that spawn without a shield. Shield ones get a shield that's their 'bonus'.